acstools/acscteforwardmodel.py

```diff
@@ -8,6 +8,10 @@
 `ACS Data Handbook <http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/>`_
 for more details.
 
+For guidance on running the CTE forward model, see the Jupyter notebook `ACS \
```

Comment:
No need for trailing slash
```suggestion
For guidance on running the CTE forward model, see the Jupyter notebook `ACS
```